_includes/footer.html

```diff
@@ -3,21 +3,27 @@
     <div>
       <img src="/assets/images/logo-gsa.png" alt="GSA logo">
     </div>
-    <div itemscope itemtype="http://schema.org/ContactPoint">
-      <h1>Contact Us</h1>
-      <a href="mailto:18f@gsa.gov" itemprop="email"><i class="fa fa-envelope"></i> 18F@gsa.gov</a>
+    <div>
+      <h1>Work</h1>
+      <a href="/consulting">Consulting</a>
+      </br>
+      <a href="/delivery">Delivery</a>
+      </br>
+      <a href="/dashboard">Dashboard</a>
       <br/>
-      <a href="https://twitter.com/18F"><i class="fa fa-twitter"></i> @18F</a>
+      <a href="https://github.com/18F"><i class="fa fa-github"></i> github.com/18F</a>
     </div>
     <div>
-      <h1>Fork Our Code</h1>
-      <a href="https://github.com/18F"><i class="fa fa-github"></i> github.com/18F</a>
-      <br/>
-      <a href="https://github.com/presidential-innovation-fellows"><i class="fa fa-github"></i> github.com/presidential-innovation-fellows</a>
-      <br/>
-      <a href="https://github.com/18F/18f.gsa.gov/edit/staging/{{ page.path }}">Edit this page</a>
-      <br/>
-      <a href="/developer/">Developer resources</a>
+      <h1> People </h1>
+      <a href="/team">Team</a>
+    </br>
```

Comment:
Although I technically think using a <br> to move the links to new lines is fine, a better approach to make things work responsively would be to use CSS. So rather the using <br> tags between each tags, you can use a class like `.footer_nav` and use display: block for each link

```css
.footer_nav a {
  display: block
}
```

Again, I'm not sure this is necessary for based on what this should do at different screen sizes, so consider that when making a change.